### PR TITLE
Add 55 changes including separate section in URL scheme

### LINKED ISSUE #506: Add new URL scheme for running a plugin command to documentation
```
sketch://plugin/my.plugin.identifier/my.command.identifier
```

### PR DESCRIPTION
**Pull request by christianklotz**
Closes #506 and #507

### CODE REVIEW

File: docs/pages/app.md [hunk 1]
```diff
@@ -33,3 +35,54 @@ document.selectedLayers.forEach(layer => console.log(layer.id))
...
+#### Example
+
+The command must implement the `HandleURL` action and have it defined in the plugin manifest in order to be run.
+
+##### `command.js`
+
+```js
+const sketch = require('sketch')
+
+function handleURL(context) {
```
Comment (mathieudutour) [inline docs/pages/app.md]: a bit tricky here: if using skpm, this will not work, it needs to be `export function handleURL...`

Comment (christianklotz) [reply docs/pages/app.md]: Hmm, yeah … I stumbled into this, too. Perhaps I just add a comment above?

```js
// export function handleURL(context) if you're using skpm
```

Comment (mathieudutour) [reply docs/pages/app.md]: sounds good

File: docs/pages/plugins-releases-55.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+})
+```
+
+### Some better logging of the prototype of wrapped objects
+
+[_GitHub pull request_](https://github.com/BohemianCoding/SketchAPI/pull/451)
+
+There was a bug in the `util.inspect` algorithm (which console uses) that makes it think the prototype of a wrapped object is a wrapped object and uses the code path for wrapped object. This has been fixed for better logging output.
+
+### Upcoming Additions
```
Comment (mathieudutour) [inline docs/pages/plugins-releases-55.md]: not sure we should put this here, it will be for 56 ;)

Comment (christianklotz) [reply docs/pages/plugins-releases-55.md]: Yeah, ok. Let's take it out

**Pull request merged**